src/com/gfredericks/test/chuck/generators.cljc

```diff
@@ -78,7 +82,8 @@
               (let [pairs (core/partition 2 v1)
                     names (map first pairs)
                     gens (map second pairs)]
-                `(for [[~@names] (gen/tuple ~@gens)
+                `(for [[~@names]
+                       (~'gen/tuple ~@gens)
```

Comment:
I'm still a bit confused about how cljs macros fit in everywhere, but my guess is that this is dangerous -- does it not assume that every namespace that calls this macro has done `(:require [clojure.test.check.generators :as gen])`?

I can see how it would be harder to do generally though, because the generators namespace is different on the different platforms, is that right? If that's the biggest blocker, I've plans to rename the cljs namespaces in test.check, so I could expidite that.